errors.go

```diff
@@ -31,10 +31,12 @@ var (
 	VolumeInvalidSizeError                  = constError("VolumeInvalidSizeError")
 
 	DatabaseAccountDestroyError      = constError("DatabaseAccountDestroyError")
-	DatabaseAccountNotFoundError     = constError("DatabaseAccountNotFoundError")
 	DatabaseAccountAccessDeniedError = constError("DatabaseAccountAccessDeniedError")
 	DatabaseCreatingAccountError     = constError("DatabaseCreatingAccountError")
 
+	// Invalid API Key Error
+	InvalidAPIKeyError = constError("InvalidAPIKeyError")
```

Comment:
@uzaxirr don't you think that this could be handled directly from the backend, improving the returning error?

Reply:
@fulviodenza That would be the best case and won't require handling from cli. 
@uzaxirr Let me what what do you think about this?
Let me know in case you guys want to send a specific error from backend, I can accommodate for that change in this PR just in case.